### ISSUE TITLE
Missing arcane missiles

### ISSUE THREAD

**Issue by Jekenius** [Atlantiss/NetherwingBugtracker#6194]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: Sometimes arcane missiles last missile is lost for some reason. I am using imp arcane missiles talent at moment and sometimes when struck last missile is lost.
![WoWScrnShot_080419_125935](https://user-images.githubusercontent.com/45003723/62420226-f14b6a80-b6b8-11e9-9cc6-cfc1eb67a5b4.jpg)
that is what talent says

**Current behaviour**:
![WoWScrnShot_080419_125859](https://user-images.githubusercontent.com/45003723/62420229-f90b0f00-b6b8-11e9-99f3-7cf29b701fa2.jpg)
That is example i screenshotted but that happened more than once. I used 2 full arcane missiles on that mob(didnt move or cast another spell during channeling time) so there should be 2*5=10 missiles, but there is only 9. Missiles were 1st and 2nd spells i used on this mob so there was no interruption due to mob's death. Only after i used missiles i used some instant spells to kill mob down.

Update: just visited dungeon and lost some last missiles even while not struck.

**Expected behaviour**: Missiles shouldnt be lost.

**Server Revision**:

**Comment by Dekirayne** [comment]
can confirm same thing happens to me

**Comment by arcwizard** [comment]
It happens on Karazhan right now, with arcane missiles Rank 2. From 4 missile somethime one missile is lost.

Server Rev: 3486.

**Comment by Tereneckla** [comment]
Duplicate of #5413
